src/ngamsPlugIns/ngamsPlugIns/ngamsCmd_MIRRTABLE.py

```diff
@@ -677,6 +678,14 @@ def assign_files_to_target_nodes(files, assigner, volumes_to_files):
                     target_host, target_volume)
         volumes_to_files.setdefault((target_host, target_volume), []).append(next_file_to_be_mirrored)
 
+def get_num_files_to_resume(iteration, ngams_server):
+    """
+    How many files in this iteration are being resumed from a failure in a previous iteration?
+    """
+    sql = "select count(file_id) from ngas_mirroring_bookkeeping where iteration = {0} and status <> 'LOCKED'"
+    num_files = ngams_server.getDb().query2(sql, args=(iteration,))[0][0]
+    logger.info('iteration %d contains %d files which will be resumed' % (iteration, num_files))
```

Comment:
Just a minor correction:
```suggestion
    logger.info('iteration %d contains %d files which will be resumed', iteration, num_files)
```